### PR TITLE
⭐️feat: Tooltip on each country in world map

### PR DESCRIPTION
**Pull request by kalvin807**
* **Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines
- [x] i18n translation has been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bugfix, feature, docs update, ..., provide screenshots if necessary)
New feature: Tooltip showing cured, death, infected data of each country when hovering/touch

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)
No

* **Other information**:
Preview:
preview

### CODE REVIEW

**Comment by kalvin807** [comment]
Some weird commit sneaked in, will fix later.

**Comment by cswbrian** [comment]
@kalvin807 lemme know when it's ready

**Comment by kalvin807** [comment]
@cswbrian  now ready

File: src/components/charts/WorldMap.js [hunk 1]
```diff
@@ -73,6 +87,25 @@ const WorldMap = ({ data }) => {
     }
   }
 
+  const ToolTipTitle = ({ props }) => {
+    const isZh = i18n.language === "zh"
+    return (
+      <>
+        <Typography color="inherit" variant="h6">
+          {" "}
+          {props.emoji} {isZh ? props.name_zh : props.name_en}
```
Comment (cswbrian) [inline src/components/charts/WorldMap.js]: Please use `withLanguage(i18n, props, "name")`

File: src/components/charts/WorldMap.js [hunk 2]
```diff
@@ -73,6 +87,25 @@ const WorldMap = ({ data }) => {
...
+        </Typography>
+        {`${isZh ? "確診" : "Confirmed"}: ${
+          props.confirmed === 0 ? "-" : props.confirmed
+        } `}
+        {`${isZh ? "死亡" : "Dead"}: ${props.died === 0 ? "-" : props.died} `}
+        {`${isZh ? "康復" : "Cured"}: ${
```
Comment (cswbrian) [inline src/components/charts/WorldMap.js]: Here are our translations
```
  "cases.status_hospitalised": "Hospitalised",
  "cases.status_discharged": "Discharged",
  "cases.status_serious": "Serious",
  "cases.status_critical": "Critical",
  "cases.status_deceased": "Deceased",
```

**Comment by elvinyhlee** [comment]
Is time to rethink whether we should continue using bubble, cause the map start getting messy now.

image

**Comment by kalvin807** [comment]
I tried to add zoom feature but the circles do not scale well. How about using colour intensity to show the data.
But I think it is a large rework so maybe better to open an issue to have some discussion.

**Comment by kalvin807** [comment]
@cswbrian Please check

**Comment by cswbrian** [comment]
@kalvin807 I just checked in my android device there's no tooltip when a map tile is clicked, is that expected?

**Comment by cswbrian** [comment]
Btw I prefer to work it one by one, let's focus on the toottip first

**Comment by kalvin807** [comment]
@cswbrian I tested it on ios and work okay, but not very responsive. Now I made it more responsive, so please check.

**Comment by cswbrian** [comment]
@kalvin807 Feel free to join our tg group https://t.me/joinchat/BwXunle2ZZZVp5KkPITO9Q

**Pull request merged**